README.md

```diff
@@ -1,15 +1,106 @@
-# react-lib-template
-A template that aims to make the implementation of `React` component packages easier and more methodic.
+# react-svg
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/react-svg
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-svg.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-svg.svg
+[build-status-url]:https://github.com/moxystudio/react-svg/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-svg/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-svg
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-svg/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-svg
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-svg.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-svg?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-svg.svg
+
+A component that renders the contents of an `SVG` file.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-svg
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+In the course of making a project, there is often a need to inject `SVG` into the DOM. This package aims to simplify this matter, allowing for `SVG` inlining and `SVG` spriting.
 
 ## Usage
 
-This repo serves as the template for the creation of MOXY's base `React` components. To use this, just select `@moxystudio/react-lib-template` as the template when creating a new repo for your new package, and you're all set to start working.
+```js
+import React from 'react';
+import Svg from '@moxy/react-svg';
+
+import styles from './index.module.css';
+
+const arrowLeft = import(/* webpackChunkName: "svg-sprite" */ '../media/icons/arrow-left.inline.svg');
+
+const Home = () => {
+    const svgPromise = Promise.resolve({ default: arrowLeft });
```

Comment:
There is no need for this. While dynamically importing we already get a Promise.

Reply:
See your demo home page as an example.